model/src/test/kotlin/config/ExcludesTest.kt

```diff
@@ -73,7 +73,7 @@ class ExcludesTest : WordSpec() {
             repository = Repository.EMPTY,
             analyzer = AnalyzerRun(
                 environment = Environment(),
-                config = AnalyzerConfiguration(allowDynamicVersions = false),
+                config = AnalyzerConfiguration(),
```

Comment:
There is one more place in `RequirementsCommand`, mind simplifying that too as part of this commit?